conformance.adoc

```diff
@@ -303,22 +288,18 @@ grid_mapping_name[: coord_var [coord_var ...]] [grid_mapping_name: [coord_var ..
 * Each coord_var is the name of a coordinate variable or auxiliary coordinate variable, which must exist in the file.
 If it is an auxiliary coordinate variable, it must be listed in the coordinates attribute.
 
-
-* The grid mapping variables must have the **`grid_mapping_name`** attribute.
+* The grid mapping variables must have the **`grid_mapping_name`** attribute. 
```

Comment:
As a general comment, it seems weird that all of the monospaced items in this section are also bold, whereas elsewhere in the document we have either/or, not both bold and monospace. Is there a reason for this?

Reply:
Ah. I suspect this is just the consequence of not having a style guide ...